bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties

```diff
@@ -1,6 +1,6 @@
-# Binding
-icloud.binding.name=iCloud Binding
-icloud.binding.description=The Apple iCloud is used to retrieve data such as the battery level or current location of one or multiple Apple devices connected to an iCloud account. Updates are quick and accurate without significant battery time impact.
+# add-on
+icloud.addon.name=iCloud Binding
```

Comment:
You probably need to look at this one? I have a feeling that there should not be an 'iCloud' prefix on these lines. Or??

Reply:
In addon.xml:
```xml
	<name>@text/icloud.binding.name</name>
	<description>@text/icloud.binding.description</description>
```

So good catch, it would probably be better to fix addon.xml to refer to icloud.addon.*

Reply:
Why is that defined in this way at all? My understanding was that this is replaced automatically.

Reply:
My guess would be to avoid redundancy, but it seems strange to do this when we have the i18m tool.